test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitControllerSpec.scala

```diff
@@ -135,6 +136,11 @@ class CheckAllAnswersAndSubmitControllerSpec
           checkPageIsDisplayed(
             performAction(),
             messageFromMessageKey("checkAllAnswers.title"), { doc =>
+              (new AcquisitionDetailsControllerSpec).validateAcquisitionDetailsCheckYourAnswersPage(
```

Comment:
if you stick `validateAcquisitionDetailsCheckYourAnswersPage` in the companion object of `AcquisitionDetailsControllerSpec` you wouldn't have to create an instance of the class